docs/building.md

```diff
@@ -158,6 +162,12 @@ cd ..
 tar -cvf air_tools.tar.gz install-aarch64
 ```
 
+## Running on the VCK5000
+
+The first step required for running on the VCK5000 is building the hardware platform and loading it on your device. [Here](../platforms/xilinx_vck5000_air) we provide documentation on how to do that. Then, the AIR tools must be installed by running the [Building external projects on X86](#building-external-projects-on-X86) followed by the [Building on x86 with runtime for PCIe](#building-on-x86-with-runtime-for-PCIe) and [Environment setup](#environment-setup). Then, the [AIR PCIe driver](../driver) must be compiled and installed. Now you are ready to run tests using AIR on the VCK5000! For example, go to [Test 13](../test/13_mb_add_one) and run `Make` followed by `sudo ./test.elf` to run the application.
+
```

Comment:
'make' should be lowercase

Reply:
Thanks for the catch. Addressed in [b86101c](https://github.com/Xilinx/mlir-air/pull/250/commits/b86101c13f422a0484b17d933b6a2699fd8db9a4).